libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp

```diff
@@ -50,8 +50,8 @@ void AP_InertialSensor::Write_IMU_instance(const uint64_t time_us, const uint8_t
         accel_x : accel.x,
         accel_y : accel.y,
         accel_z : accel.z,
-        gyro_error  : get_gyro_error_count(imu_instance),
-        accel_error : get_accel_error_count(imu_instance),
+        gyro_error  : _gyro_error_count[imu_instance],
```

Comment:
The inline function costs nothing. I think this makes it a little less readable / understandable - but that's just me